### DISCUSSION TITLE
You can set the tangential velocity of air supply in fds, how to set the tangential velocity of smoke exhaust?

### DISCUSSION THREAD

**Discussion by ZXJIN119** [firemodels/fds#13831] [Q&A] [answered]
You can set the tangential velocity of air supply in fds, how to set the tangential velocity of smoke exhaust?
<img width="390" alt="401e4f163a2e8c8e3bda5da2649422c" src="https://github.com/user-attachments/assets/f0da4665-52a3-41d0-960c-fcab2903edbb">

**Comment by drjfloyd** [comment]
You cannot. This isn't something that happens to any real extent in reality except very close to a louver..

**Reply by ZXJIN119** [reply]
Is there any way to set the angle of smoke evacuation, such as setting the direction of the vent, or tangential speed or something?

**Reply by drjfloyd** [reply, marked as answer]
You can set the face of the wall cell that the vent is on, but tangential flow at an exhaust doesn't really happen except very close to an exhaust vent.  Air moves from high pressure to low pressure. For supply the high pressure is in the duct and the air is constrained to moving along the duct. So you have a uniform direction of flow hitting louvers that gets redirected.  For exhaust the high pressure is outside. Air can move from any direction towards the low pressure behind the louver. It isn't until the air reaches the louver that it is redirected.

![image](https://github.com/user-attachments/assets/502f41ae-1a8a-4ce0-a34a-b2a998643ad8)

**Reply by ZXJIN119** [reply]
Thank you very much for the help you have provided, decided to try the following method in order to show the difference between the angles of the different smoke vents (similar to shutters).
![图片2](https://github.com/user-attachments/assets/fa80d331-6d5c-40a6-bd42-4cae0b29fcb2)
